hirundo/cli.py

```diff
@@ -32,18 +37,30 @@ def upsert_env(var_name: str, var_value: str):
         var_name: The name of the environment variable to change.
         var_value: The new value of the environment variable.
     """
-    dotenv = "./.env"
     regex = re.compile(rf"^{var_name}=.*$")
-    with open(dotenv) as f:
-        lines = f.readlines()
+    lines = []
+    if os.path.exists(dotenv_filepath):
+        with open(dotenv_filepath) as f:
+            lines = f.readlines()
 
-    with open(dotenv, "w") as f:
+    with open(dotenv_filepath, "w") as f:
         f.writelines(line for line in lines if not regex.search(line) and line != "\n")
 
-    with open(dotenv, "a") as f:
+    with open(dotenv_filepath, "a") as f:
         f.writelines(f"\n{var_name}={var_value}")
 
 
+def upsert_env(var_name: str, var_value: str):
+    if os.path.exists(EnvLocation.DOTENV.value):
+        # If a `.env` file exists, re-use it
+        _upsert_env(EnvLocation.DOTENV.value, var_name, var_value)
+        return EnvLocation.DOTENV.name
+    else:
```

Comment:
What if the `.hirundo.conf` file doesn't exist?

Reply:
The file will be created